e2eshark/run.py

```diff
@@ -1157,9 +1148,13 @@ def main():
         help="Please select a dir with large free space to cache all torch, hf, turbine_tank model data",
         required=True,
     )
-    
+
     args = parser.parse_args()
     cache_dir = args.cachedir
+    if not os.path.exists(cache_dir):
```

Comment:
I have this in other PR. Maybe remove this so we have everything related to cache_dir documented there?